sample-scenarios/sample4/README.md

```diff
@@ -200,9 +200,9 @@ Further it will invoke the get with the scope and it will return the response as
 
 ### References
 
-[1] - [https://docs.wso2.com/display/AM2xx/Managing+Public%2C+Partner+vs+Private+APIs+Sample+Documentation](https://docs.wso2.com/display/AM2xx/Managing+Public%2C+Partner+vs+Private+APIs+Sample+Documentation)
+[1] - [https://docs.wso2.com/display/AM220/Managing+APIs](https://docs.wso2.com/display/AM220/Managing+APIs)
```

Comment:
This should be "AM250", isn't it?

Reply:
We don't have docs for 2.5.0 yet. So adding the last GA release.

Reply:
No I think we have to put AM250. Otherwise when we release docs it should be a problem.